src/Mobs/Path.cpp

```diff
@@ -51,6 +51,14 @@ cPath::cPath(
 	m_Source.x = FloorC(a_StartingPoint.x - HalfWidthInt);
 	m_Source.y = FloorC(a_StartingPoint.y);
 	m_Source.z = FloorC(a_StartingPoint.z - HalfWidthInt);
+	
+	if ((m_Source.y <= 0) || (m_Destination.y <= 0)
+		|| (m_Source.y > 256) || (m_Destination.y > 256))
```

Comment:
One option is to treat outside the world as always empty. For the bottom it might try to path in to holes, but give up when it can't find a ground to walk on.